_posts/en/2020-05-20-Team-Topologies-Whitelisting-app.md

```diff
@@ -0,0 +1,85 @@
+
+The last note, is that while these timelines are ahead of what is typically found in government environments, it is only a starting point. My personal goal has always been to match Google's ninety minutes to create a prototype for Google Glass (reference [How Google Works](https://www.goodreads.com/book/show/23158207-how-google-works)). If we were to have demo'd the application to the CIO after ninety minutes of having started work, the only remaining question would be to how to get that number down to 60 minutes. Never stop improving.
+
+## Model
+
+The methodology outlined throughout this document is heavily borrowed from [Team Topologies](https://teamtopologies.com/?gclid=EAIaIQobChMI4N2R7fzM6AIV1f_jBx2o3ACIEAAYASAAEgL8DPD_BwE).
+
+![Compare curves]({{ site.baseurl }}/assets/images/TTBlog-Figure1.PNG)
+_Figure 1_
+
+In a government context, the management model looked as follows
+
+![Compare curves]({{ site.baseurl }}/assets/images/TTBlog-Figure2.PNG)
+_Figure 2_
+
+It should be noted that in our case, the separation between the DTS platform team and the IT Strategy team was not as well defined as it appears here. For example, throughout the project the DTS resources attended the same standup meetings as the IT Strategy resources. Further, the value stream aligned team was composed of programmers from the DTS team as well. This model oversimplifies the complexities of life, of course. The reason why we have represented the interactions between the teams above is twofold. Firstly, to represent how responsibilities would be split between teams if this solution were to scale. In our case, the application was simple enough, and the team small enough, to permit for a mixing of resources as we did. As one scales, this becomes less feasible (keeping in mind we want to reduce the cognitive load on team members, and keep teams as small as possible, while still being able to deliver the product). The second reason the interactions have been represented this way, is to communicate how the technologies interacted. In between the teams we see XaaS, because this is the terminology used in the book, Team Topologies, that I am heavily leaning upon. I have recently begun to use a different term -- termination point. That is required is a termination point of responsibilities. In the case of our team, it was the termination point between members of the team, in large projects, it would be the termination point between different teams. A termination point is where the responsibilities of one group are transferred to another.
```

Comment:
> Further , the value stream 

Further more, the value stream

> This model oversimplifies the complexities of life, of course 

Recommend removing or rewording as it doesn't really add to the paragraph 

> XaaS

define this abbreviation 

>  I have recently begun to use a different term -- termination point. That is required is a termination point of responsibilities.

not making much sense to me... might just be me running on little sleep 

> A termination point is where the responsibilities of one group are transferred to another.

Should be placed after you initially specified the term